.idea/misc.xml

```diff
@@ -1,9 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="false" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
Don't change this manually but find some time to make sure that your Android Studio is up to date along with your Java Development Kit (JDK). You went down a version in this commit, but that's okay for now.